person_class.rb

```diff
@@ -23,6 +23,10 @@ def correct_name
     @name
   end
 
+  def add_rental(date, book)
+    @rentals.push(Rental.new(date, book, self))
```

Comment:
- [ ] After the `person.rentals << self` code we added earlier in the rental class, simply creating a rental already pushes the rentals in the array. Kindly modify this method to be similar to the one in the book class.

  ```
  Rental.new(date, book, self)
  ```